### DISCUSSION TITLE
Candler School of Theology 2020-21 Catalog and Handbook

### DISCUSSION THREAD

**Discussion by swroberts** [thegetty/quire#64] [Show & Tell]
[Pitts Theology Library](http://pitts.emory.edu) and [Candler School of Theology](http://candler.emory.edu) at [Emory University](http://emory.edu) collaborated to produce the [2020-2021 Catalog and Handbook](http://catalog.candler.emory.edu), containing the academic calendar, program descriptions, academic policies, course descriptions, lists of personnel, and resources for students. Although our use of Quire doesn't fall into the usual categories, we realized that Quire offered a unique opportunity to create a searchable, easily navigated website with a matching downloadable PDF format that could be easily updated from year to year with an [archive of previous years](https://catalog.candler.emory.edu/archive/) for reference. We will publish a new version of the site each year as the content is updated.

Our implementation of Quire involves moderate adjustments to the starter theme, including changes to the banner, navigation methods, color schemes, footer contents, and so on. Our theme is not particularly suited for other uses because it was designed specifically for the Emory brand, but if you are interested for experimenting, I'm happy to share it. Our publishing process involved staff at Candler doing content updates to previous catalogs and staff at Pitts doing initial content migration, Quire setup/customization, and project management. We host the site on a [Reclaim Hosting](https://reclaimhosting.com/) shared server.

**Comment by geealbers** [comment]
Thanks so much for sharing this @swroberts! It's great to see a whole new use case for Quire and I have to say I was tickled to click through and not even recognize it immediately as a Quire project. I'm glad you were able to customize it like you did. Thanks too for sharing some of your publishing process and being willing to share the customization work you did on it with other Quire users. That's terrific!
